torch/_functorch/partitioners.py

```diff
@@ -78,7 +78,7 @@ def _extract_graph_with_inputs_outputs(joint_graph, inputs, outputs):
             continue
         elif node.op == 'placeholder':
             env[node] = InvalidNode
-        elif node.op == 'call_function':
+        elif node.op == 'call_function' or node.op == 'call_method':
```

Comment:
Is this because the new higher order op exposes its inner function (the backward hook) as a call_method?


For a second I was worried that the partitioner seeing an opaque blob would mean it might make a poor partitioning decision, but I realized that's wrong - we know that the backward hook blobs **always** have to go in the backward graph anyway.

Reply:
I dont know if we need this anymore, but yes, this is the reason.